types/knuth-shuffle/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for knuth-shuffle 1.0
+// Project: https://git.coolaj86.com/coolaj86/knuth-shuffle.js
+// Definitions by: Chris Atkin <https://github.com/atkinchris>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function knuthShuffle(array: any[]): void;
```

Comment:
The function returns the input array. Is the return type `void` an intentional, ah, improvement of the source API?

Reply:
You're very right - this was unintentional! It mutates the passed array, so I've never used the return - I'll correct the typings.